backend/src/main/java/com/woowacourse/edd/application/service/VideoService.java

```diff
@@ -0,0 +1,56 @@
+package com.woowacourse.edd.application.service;
+
+import com.woowacourse.edd.application.converter.VideoConverter;
+import com.woowacourse.edd.application.dto.VideoPreviewResponse;
+import com.woowacourse.edd.application.dto.VideoSaveRequestDto;
+import com.woowacourse.edd.application.response.VideoResponse;
+import com.woowacourse.edd.domain.Video;
+import com.woowacourse.edd.exceptions.VideoNotFoundException;
+import com.woowacourse.edd.repository.VideoRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class VideoService {
+
+    private static final String CREATE_DATE = "createDate";
+    private final VideoRepository videoRepository;
+    private final VideoConverter videoConverter;
+
+    @Autowired
+    public VideoService(VideoRepository videoRepository, VideoConverter videoConverter) {
+        this.videoRepository = videoRepository;
+        this.videoConverter = videoConverter;
+    }
+
+    public VideoResponse save(VideoSaveRequestDto requestDto) {
+        Video video = videoRepository.save(videoConverter.toEntity(requestDto));
+
+        return videoConverter.toResponse(video);
+    }
+
+    public List<VideoPreviewResponse> findVideosByDate(int page, int limit) {
+        PageRequest pageRequest = PageRequest.of(page, limit, Sort.by(CREATE_DATE).descending());
+        Page<Video> foundVideos = videoRepository.findAll(pageRequest);
+
+        return foundVideos.getContent().stream()
```

Comment:
```suggestion
        return foundVideos.map(videoConverter::toPreviewResponse)
                .getContent();
```

응답을 Page<VideoPreviewResponse>로 변경하면 getContent도 필요없을거에요

Reply:
Page로 변경해서 getcontent 삭제했습니다.